### PR TITLE
Added Proxy App tag

### PR DESCRIPTION
**Pull request by akhilpatlolla**

### CODE REVIEW

File: var/spack/repos/builtin/packages/kripke/package.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 # LLNL-CODE-647188
 #
 # For details, see https://github.com/llnl/spack
-# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+# Please also see the LICENSE file for our notice and the LGPL.
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/kripke/package.py]: You probably shouldn't be making any changes to the copyright.

File: var/spack/repos/builtin/packages/kripke/package.py [hunk 2]
```diff
@@ -25,17 +25,22 @@
...
     homepage = "https://codesign.llnl.gov/kripke.php"
     url      = "https://codesign.llnl.gov/downloads/kripke-openmp-1.1.tar.gz"
 
+    tags = ['proxy-app']
     version('1.1', '7fe6f2b26ed983a6ce5495ab701f85bf')
 
-    variant('mpi',    default=True, description='Build with MPI.')
-    variant('openmp', default=True, description='Build with OpenMP enabled.')
+    variant('mpi', default=True, description='Build with MPI')
+    variant('openmp', default=True, description='Build with with OpenMP')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/kripke/package.py]: double with

**Comment by akhilpatlolla** [comment]
Not sure y it failed!
` 11645.3  Python: 2.7  TEST_SUITE=build COVERAGE=true SPEC=hypre^mpich  49 min 56 sec`

**Comment by adamjstewart** [comment]
I restarted the job, I don't think it's related to your PR.

**Pull request merged**